swaglyrics/cli.py

```diff
@@ -65,24 +67,26 @@ def get_lyrics(song, artist):
 	:return: song lyrics or None if lyrics unavailable
 	"""
 	url_data = stripper(song, artist)  # generate url path using stripper()
-	url = f'https://genius.com/{url_data}-lyrics'  # format the url with the url path
-	try:
-		page = requests.get(url)
-		page.raise_for_status()
-	except requests.exceptions.HTTPError:
-		url_data = requests.get(f'{backend_url}/stripper', data={
-			'song': song,
-			'artist': artist}).text
-		if not url_data:
-			return None
-		url = 'https://genius.com/{}-lyrics'.format(url_data)
-		page = requests.get(url)
-
-	html = BeautifulSoup(page.text, "html.parser")
-	lyrics_path = html.find("div", class_="lyrics")  # finding div on Genius containing the lyrics
-	lyrics = UnicodeDammit(lyrics_path.get_text().strip()).unicode_markup
-	return lyrics
-
+	if not url_data: 		
+		return None
+	else:
+		url = f'https://genius.com/{url_data}-lyrics'  # format the url with the url path
+		try:
```

Comment:
The function stops execution once a return statement is executed so you do not need the else.

Reply:
but if it doesn't meet the if the condition doesn't it then run else?

Reply:
```python
def f(x):
    if x > 5:
        return 23
    print('a')
```
def f(x):
    if x > 5:
        return 23
    else:
        print('a')
```
Both will do the same thing ;)

Reply:
ohhhhhhh ok, I get what you mean

Reply:
fixed